authors/apps/profiles/serializers.py

```diff
@@ -0,0 +1,36 @@
+from .models import UserProfile
+from rest_framework import serializers
+
+image = 'http://correctimage.png'
```

Comment:
Why are we using this?

Reply:
We wanted to have like a default avatar, but pylint kept complaining that it was long, but we have removed that part